pages/index.tsx

```diff
@@ -138,3 +137,34 @@ export default function Donate({
     </>
   );
 }
+
+export async function getServerSideProps({ locale }) {
```

Comment:
We also create a static export of this application for hosting at Netlify (for some tenants) with `npm run build && npm run export` - I am not sure if this static hosting supports `getServerSideProps` as this is listed as unsupported at https://nextjs.org/docs/advanced-features/static-html-export.

Reply:
Maybe we want to change from a static HTML export towards using server-side rendering. It seems to be possible reading https://docs.netlify.com/integrations/frameworks/#next-js.

Reply:
One more question: This pull request is using `getServerSideProps`, but also `getStaticProps`. Is the usage of `getServerSideProps` required?

Reply:
> Two things not working for me regarding language switching:
> 
> * [ ]  The page is now using a locale in the path name (not sure how this got activated, but we had not used it before), e.g. https://planet-webapp-iag7azjnz-planetapp.vercel.app/de/plant-for-ethiopia instead of https://planet-webapp-iag7azjnz-planetapp.vercel.app/plant-for-ethiopia - this seems to override any chosen language from the language switcher in the UI. The UI language switch should also change the locale in the path or we should remove using any locale in the path again.
> * [ ]  If you change the language by the language switcher in the UI (either in the small footer on the home page or using the switcher in the profile being logged in), the newly chosen language does not seem to get stored permanently. Once you load another page it is shown in the formerly chosen language again.

I will look into it

Reply:
Is `getServerSideProps` is still left in `pages/open-app.tsx` intentionally?

Reply:
> Is `getServerSideProps` still left in `pages/open-app.tsx` intentionally?

I think I wrote this by mistake, I'll replace this  with `getStaticProp`